service/src/structure_comparer/serve.py

```diff
@@ -798,6 +799,46 @@ async def get_mapping(
         return ErrorModel.from_except(e)
 
 
+@app.get(
+    "/project/{project_key}/mapping/{mapping_id}/results",
```

Comment:
I think it would be nicer to have type of representation in the URL rather than just "result". This would allow to also have a StructureMap "result".
```suggestion
    "/project/{project_key}/mapping/{mapping_id}/html",
```